packages/office-ui-fabric-react/src/components/Image/Image.tsx

```diff
@@ -137,11 +138,9 @@ export class Image extends React.Component<IImageProps, IImageState> {
   private _computeCoverStyle(props: IImageProps) {
     let { imageFit } = props;
     if (imageFit === ImageFit.cover || imageFit === ImageFit.contain) {
-      let { image } = this.refs;
+      let { image, frame } = this.refs;
       if (image) {
-        let { width, height } = props;
-
-        let desiredRatio = (width as number) / (height as number);
+        let desiredRatio = frame.clientWidth / frame.clientHeight;
```

Comment:
hmm. This is measuring dimensions for every image which is a performance concern.

Most of the time, the user knows the width and height they want and can just pass it in without reading it from the browser, so this should only be done in an opt in mode I think. I'd be ok with